packages/ui/src/ui/pages/query-tracker/QueryTrackerTopRow/QuerySelector.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {ReactElement} from 'react';
+import {Select, SelectProps} from '@gravity-ui/uikit';
+import type {Option} from '@gravity-ui/uikit/build/esm/components/Select/tech-components';
+import {SelectOption} from '@gravity-ui/uikit/build/esm/components/Select/types';
+import './QuerySelector.scss';
+import cn from 'bem-cn-lite';
+
+const controlBlock = cn('yt-query-selector-control');
+const popupBlock = cn('yt-query-selector-popup');
+
+type Props<T> = {
+    items: T[];
+    value?: string;
+    onChange: (value: string) => void;
+    children: (items: T[]) => ReactElement<SelectOption<T>, typeof Option>[];
+} & Omit<SelectProps, 'children' | 'onUpdate' | 'value'>;
```

Comment:
There is no reason to get rid of `onUpdate` property, sometimes we need it if we use a component from `DFDialog`.

Reply:
I am removing this method due to the fact that I am overriding it below.